jest.config.js

```diff
@@ -10,4 +24,7 @@ module.exports = createConfig('jest', {
     'src/setupTest.js',
     'src/i18n',
   ],
+  moduleNameMapper: {
+    'env.config': envConfigPath,
+  },
 });
```

Comment:
Admittedly, once I started diving into the jest config and tracking down what it came with out-of-the-box from `createConfig('jest')`, I wasn't sure if the JS config handling in `frontend-build` (as you linked) was actually _working_ because it still wasn't picking up on the fact that `env.config.js` existed and needed to be handled in these tests. 
So two questions: 
1. Should we assume that the `jest.config.js` in `frontend-build` should have added the environment variables from an `env.config.js` to the ConfigDocument? 
2. If not, would my use of `mergeConfig()` in this repo's `setupTest.js` be appropriate to bring upstream to `frontend-build`'s [setupTest.js file](https://github.com/openedx/frontend-build/blob/35415fe423a6f1103eacf1b761c003a670537726/config/jest/setupTest.js)?

Reply:
I will add, not having the `moduleNameMapper` logic in this file didn't break any tests, but I left it there out of paranoia and due to having stared at the screen for too long this week, so I'm very likely to remove it for this PR.